global.json

```diff
@@ -1,5 +1,5 @@
 {
     "sdk": {
-        "version": "6.0.400"
+        "version": "8"
```

Comment:
The GeneticSharp is officially only supporting .NET 6, so it's not a good idea to change it here.

When I have some time available, I will make a PR to support .NET 9.